packages/syft/src/syft/service/user/user_service.py

```diff
@@ -84,6 +91,212 @@ def create(
         user = result.ok()
         return user.to(UserView)
 
+    @service_method(
+        path="user.forgot_password", name="forgot_password", roles=GUEST_ROLE_LEVEL
+    )
+    def forgot_password(
+        self, context: AuthedServiceContext, email: str
+    ) -> SyftSuccess | SyftError:
+        result = self.stash.get_by_email(credentials=context.credentials, email=email)
+        # Isn't a valid email
+        if result.is_err():
+            return SyftSuccess(
```

Comment:
1. Is this a success because we don't want someone to be able to infer if someone is a member of the server or not by spamming? 
2. It looks like we reuse the same message for the SyftSuccess each time so maybe we can just have that syft success as a var and return that so the messages don't drift (the last message is slightly different from this one).

Reply:
1 - Yup
2 - Good catch! I've added a new variable on top of this service and used it everywhere.